### PR TITLE
Basic info in User Profile (#44)

### PR DESCRIPTION
**Pull request by jimmeak**

### CODE REVIEW

File: ddcz/models/used/users.py [hunk 1]
```diff
@@ -103,6 +157,20 @@ def profile_url(self):
             kwargs={"user_profile_id": self.pk, "nick_slug": self.slug},
         )
 
+    @property
+    def registration_date(self):
```
Comment (Almad) [inline ddcz/models/used/users.py:161]: I'd argue that this should either have `_str` suffix or be handed in the template. When I am retrieving `something_date` from model, I expect a date/time-like object, IMHO. 

I'd keep it consistent as with creations—we also use a filter there.

File: ddcz/templates/users/detail.html [hunk 2]
```diff
@@ -1,12 +1,139 @@
...
-    {% endif %}
-    <p><span class="label">Popis:</span> <span class="value">{{ profile.popis_uzivatele }}</span></p>
+
+
+    <div id="user_container">
+        <section id="user_info">
+            <h3>Info o uživatel{% if profile.is_female %}ce{%else%}i{% endif %}</h3>
+            <table>
+                <tr>
+                    <th>ID</th>
```
Comment (Almad) [inline ddcz/templates/users/detail.html:16]: 🤔 ...are we going to display it? :)

File: ddcz/templates/users/detail.html [hunk 3]
```diff
@@ -1,12 +1,139 @@
...
+                    </td>
+                </tr>
+                {% if profile.id == user.profile.id %}
+                <tr>
+                    <th>Level</th>
+                    <td><p>{{ level_description }}</p></td>
+                </tr>
+                {% endif %}
+                <tr>
+                    <th>Spatřen{% if profile.is_female %}a{% endif %} </th>
```
Comment (Almad) [inline ddcz/templates/users/detail.html:86]: I'd add `Naposledy spatřen/a`

File: ddcz/templates/users/detail.html [hunk 4]
```diff
@@ -1,12 +1,139 @@
...
+                </tr>
+                <tr>
+                    <th>Registrován{% if profile.is_female %}a{% endif %} </th>
+                    <td>{{ profile.registration_date }}</td>
+                </tr>
+            </table>
+        </section>
+
+
+        <section id="user_patron">
```
Comment (Almad) [inline ddcz/templates/users/detail.html:97]: Commenting out for now?

File: ddcz/views.py [hunk 5]
```diff
@@ -474,12 +476,24 @@ def users_list(request):
 @require_http_methods(["GET"])
 def user_profile(request, user_profile_id, nick_slug):
     user_profile = get_object_or_404(UserProfile, id=user_profile_id)
+    try:
+        creations = Author.objects.get(user=user_profile).get_all_creations()
+    except Author.DoesNotExist as e:
+        creations = False
+
+    description = LEVEL_DESCRIPTIONS["0"]
+    if user_profile.level in LEVEL_DESCRIPTIONS.keys():
```
Comment (Almad) [inline ddcz/views.py:485]: fwiw, you can do just `if user_profile.level in LEVEL_DESCRIPTIONS:`

**Review by Almad — CHANGES_REQUESTED** [review]
Just one thing—the strftime vs filter please

File: ddcz/models/used/users.py (hunk 1 above)
Comment (jimmeak) [reply ddcz/models/used/users.py:161]: I used the date templatetag then and I am returning the object. :)

File: ddcz/templates/users/detail.html (hunk 2 above)
Comment (jimmeak) [reply ddcz/templates/users/detail.html:16]: Of course :D Users will know it anyway from the url address and as I remember, people were happy to know their ID.

File: ddcz/views.py (hunk 5 above)
Comment (jimmeak) [reply ddcz/views.py:485]: Nice, thank you

File: ddcz/templates/users/detail.html (hunk 4 above)
Comment (jimmeak) [reply ddcz/templates/users/detail.html:97]: Commented

File: ddcz/templates/users/detail.html (hunk 2 above)
Comment (Almad) [reply ddcz/templates/users/detail.html:16]: :trollface:

**Review by Almad — APPROVED** [review]

**Pull request merged**